packages/core/src/NodeExecuteFunctions.ts

```diff
@@ -14,6 +14,7 @@
 /* eslint-disable @typescript-eslint/no-non-null-assertion */
 /* eslint-disable @typescript-eslint/no-shadow */
 /* eslint-disable no-param-reassign */
+/* eslint-disable @typescript-eslint/prefer-optional-chain */
```

Comment:
Added for build to pass. Same for `AuthController` and `PasswordResetController`.